llvm/lib/Analysis/AliasAnalysis.cpp

```diff
@@ -264,13 +265,20 @@ class AAInstrumentation {
     std::string CurrSeq =
         AASequenceString.substr(AASequenceString.find("-") + 1);
     Sequence.reserve(Len);
```

Comment:
You don't need this, the assignment in the next line will replace the underlying vector

Reply:
You can probably bake above line into the splitString function as well